src/adeft/modeling/classify.py

```diff
@@ -103,8 +103,10 @@ def __init__(self, shortforms, pos_labels, random_state=None):
         self.confusion_info = None
         self.other_metadata = None
         # Add shortforms to list of stopwords
-        self.stop = set(english_stopwords).union([sf.lower() for sf
-                                                  in self.shortforms])
+        self.stop = list(
+            set(english_stopwords).union([sf.lower() for sf
+                                          in self.shortforms])
+        )
```

Comment:
`TfidfVectorizer` now only allows lists for `stop_words` and no longer accepts generic containers.